gnuradio-runtime/examples/volk_benchmark/volk_plot.py

```diff
@@ -4,7 +4,8 @@
 from __future__ import unicode_literals
 import sys, math
 import argparse
-from volk_test_funcs import *
+from volk_test_funcs import (create_connection,
```

Comment:
It seems that the way you are adding the line breaks is different for each of your changes. It would be better if they are consistent along the whole PR. Mind formatting these changes consistently?